pulsar/pulsar.sh

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+readonly ROLE="$(/usr/share/google/get_metadata_value attributes/dataproc-role)"
+readonly CLUSTER_NAME="$(/usr/share/google/get_metadata_value attributes/dataproc-cluster-name)"
+readonly NUM_WORKERS=$(/usr/share/google/get_metadata_value attributes/dataproc-worker-count) 
+readonly ZOOKEEPER_HOME=/usr/lib/zookeeper
+
+readonly PULSAR_FUNCTIONS_ENABLED="$(/usr/share/google/get_metadata_value attributes/pulsar-functions-enabled || echo 'false')"
+readonly BUILTIN_CONNECTORS_ENABLED="$(/usr/share/google/get_metadata_value attributes/builtin-connectors-enabled || echo 'false')"
+readonly TIERED_STORAGE_OFFLOADERS_ENABLED="$(/usr/share/google/get_metadata_value attributes/tiered-storage-offloaders-enabled || echo 'false')"
+
+readonly PULSAR_VERSION="$(/usr/share/google/get_metadata_value attributes/pulsar-version || echo '2.6.0')"
```

Comment:
Add a variable PULSAR_HOME with value `/opt/apache-pulsar-${PULSAR_VERSION}`, so that users/developers could easily figure out where pulsar is installed.